src/plugins/intel_gpu/src/graph/reorg_yolo.cpp

```diff
@@ -27,6 +29,28 @@ layout reorg_yolo_inst::calc_output_layout(reorg_yolo_node const& node, kernel_i
     return layoutTemp;
 }
 
+template<typename ShapeType>
+std::vector<layout> reorg_yolo_inst::calc_output_layouts(reorg_yolo_node const& node, kernel_impl_params const& impl_param) {
+    auto desc = impl_param.typed_desc<reorg_yolo>();
+    auto input_layout = impl_param.get_input_layout(0);
+    auto output_type = desc->output_data_types[0].value_or(input_layout.data_type);
+    auto output_format = input_layout.format;
+
+    ov::op::v0::ReorgYolo op;
+    std::vector<size_t> strides { static_cast<size_t>(desc->stride) };
+    op.set_strides(strides);
+
+    std::vector<ShapeType> output_shapes = { ShapeType() };
+    std::vector<ShapeType> input_shapes = {
+        input_layout.get<ShapeType>()
+    };
+    ov::op::v0::shape_infer(&op, input_shapes, output_shapes);
```

Comment:
PR #18579 changes usage of `shape_infer` functions. Just for information that there can be conflicts after merge into master.

Reply:
I think once this PR is merged first, it seems that shape_infer API also needs to be updated again in your PR through rebase.